packages/orbit-components/src/common/consts.ts

```diff
@@ -15,3 +15,13 @@ export enum PLACEMENTS {
   LEFT_START = "left-start",
   LEFT_END = "left-end",
 }
+
+export enum SPACINGS_AFTER {
```

Comment:
There was no need to move the `tokens` from the old file because we already have them in Tailwind, right?

Reply:
Yes. On the previous file, they were used to retrieve the value of the token, so it could then be used to apply `margin-bottom`. With Tailwind, we don't need it because we just use an `mb-` class with the correct suffix that matches a valid spacing value (defined by tokens, on the `tailwind-preset-package`.

The `tailwind/spaceAfter.ts` file is responsible for matching these SPACINGS_AFTER consts  (used by the components that have `spaceAfter` prop) to the correct tailwind class — so no tokens are needed in this process.